### PR TITLE
Add web page dashboard widget

### PR DESCRIPTION
**Pull request by khmylov**

### CODE REVIEW

File: Dashboard Web Page Widget/DashboardWebPageWidget.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+tau.mashups
+    .addDependency('jQuery')
+    .addDependency('react')
+    .addDependency("tau/configurator")
```
Comment (victor-homyakov) [inline Dashboard Web Page Widget/DashboardWebPageWidget.js]: Inconsistent quotes

File: Dashboard Web Page Widget/DashboardWebPageWidget.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        function makeCb(cb) {
+            return function() {
+                cb.apply(null, _.rest(_.toArray(arguments)));
+            };
+        }
+
+        var D = React.DOM;
+        
+        var WebpageWidget = React.createClass({
+            displayName: 'tp_experimental_webpage_frame',
```
Comment (victor-homyakov) [inline Dashboard Web Page Widget/DashboardWebPageWidget.js]: I'd like to remove "experimental" from production code

File: Dashboard Web Page Widget/DashboardWebPageWidget.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
+tau.mashups
+    .addDependency('jQuery')
+    .addDependency('react')
+    .addDependency("tau/configurator")
+    .addMashup(function($, React, configurator) {
+        function makeCb(cb) {
```
Comment (victor-homyakov) [inline Dashboard Web Page Widget/DashboardWebPageWidget.js]: Generic callback creator with only one usage. Could be inlined.

File: Dashboard Web Page Widget/DashboardWebPageWidget.js (hunk 2 above)
Comment (khmylov) [inline Dashboard Web Page Widget/DashboardWebPageWidget.js]: Will rename it to `tp_webpage_frame_mashup` then, OK?

**Comment by victor-homyakov** [comment]
LGTM

**Pull request merged**